docs/configure/nodes.md

```diff
@@ -61,6 +61,10 @@ The Netdata config directory also contains one symlink:
   versions are copied into the config directory when opened with `edit-config`. _Do not edit the files in
   `/usr/lib/netdata/conf.d`, as they are overwritten by updates to the Netdata Agent._
 
+## Configure a Netdata docker container
+
+See [configure agent containers](packaging/docker/README.md#configure-agent-containers).
```

Comment:
```suggestion
See [configure agent containers](packaging/docker/README.md#configure-agent-containers).
```
```suggestion
See [configure agent containers](/packaging/docker/README.md#configure-agent-containers).
```

Should start with `/`, forgot about it

an existing example:

https://github.com/netdata/netdata/blob/d6396755b83a0931f1037f4b5de3dc7331b923fd/docs/anonymous-statistics.md?plain=1#L93